client/v3/retry_interceptor.go

```diff
@@ -59,7 +59,7 @@ func (c *Client) unaryClientInterceptor(optFuncs ...retryOption) grpc.UnaryClien
 			if lastErr == nil {
 				return nil
 			}
-			c.GetLogger().Warn(
+			c.GetLogger().Debug(
```

Comment:
This would turn of retrying logging entirely. 

How about logging it on Debug if its 1'st retry (`attempt=0`) and 
Warning if its a consecutive retry ?

Reply:
Sure, updated